src/util.js

```diff
@@ -140,6 +140,18 @@ function resolveParameterDiff(params, defaults) {
   return Object.assign(defaults, params);
 }
 
+/**
+ * gets description from query comments
+ */
+function getQueryDescription(query) {
+  const match = /^(?<=\n).*(?=\n\*)/smg.exec(query);
+
+  if (match && match.length > 0) {
+    return match[0];
+  }
+  return '';
```

Comment:
Only nitpicking: you do not have a test for that case, i.e. make sure if there is no description in the sql file that everything works as expected.